File: FaucetPay .Net FrameWork/Logs/Logs.vb
Namespace FaucetPayLogs
    Friend Class FLog
        Friend Sub Loggs(Logs As String)
            Try
                Dim Folder As String = Environment.CurrentDirectory & "\Logs"
                Dim DateFile As String = DateTime.Now.ToShortDateString.Replace("\", "-").Replace("/", "-")
                Dim DateServer As String = DateTime.Now
                Dim FileLog As String = Folder & "\Log_" & DateFile & ".log"
                Dim Message As String = DateServer & ": " & Logs


                If Not IO.Directory.Exists(Folder) Then
                    IO.Directory.CreateDirectory(Folder)
                End If

                If IO.File.Exists(FileLog) Then
                    IO.File.AppendAllText(FileLog, vbCrLf & Message)
                Else
                    IO.File.AppendAllText(FileLog, Message)
                End If


            Catch ex As Exception
            End Try

        End Sub
    End Class
End Namespace

File: FaucetPay CSharp .Net FrameWork/Logs/Logs.cs
using System;
using System.IO;
using Microsoft.VisualBasic; // Install-Package Microsoft.VisualBasic
using Microsoft.VisualBasic.CompilerServices; // Install-Package Microsoft.VisualBasic

namespace FaucetPayLogs
{
    internal partial class FLog
    {
        internal void Loggs(string Logs)
        {
            try
            {
                string Folder = Environment.CurrentDirectory + @"\Logs";
                string DateFile = DateTime.Now.ToShortDateString().Replace(@"\", "-").Replace("/", "-");
                string DateServer = Conversions.ToString(DateTime.Now);
                string FileLog = Folder + @"\Log_" + DateFile + ".log";
                string Message = DateServer + ": " + Logs;


                if (!Directory.Exists(Folder))
                {
                    Directory.CreateDirectory(Folder);
                }

                if (File.Exists(FileLog))
                {
                    File.AppendAllText(FileLog, Constants.vbCrLf + Message);
                }
                else
                {
                    File.AppendAllText(FileLog, Message);
                }
            }


            catch 
            {
            }

        }
    }
}